Sources/MessagingInApp/Gist/Managers/Models/Message.swift

```diff
@@ -144,4 +145,11 @@ extension Message: Equatable {
         // If the queueId is nil, we fallback to the instanceId which is a unique ID generated client-side
         return lhs.instanceId == rhs.instanceId
     }
+
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(instanceId)
+        hasher.combine(queueId)
+        hasher.combine(priority)
+        hasher.combine(messageId)
+    }
```

Comment:
interesting what was the use-case behind adding this? specially because I see all parameters being added into the hash, while in checking equality we are only checking two? so either we check all 4 in equality or just add 2 in here?

Reply:
Because of `messagesInQueue: Set<Message>` in `InAppMessageState` as set only allows `Hashable` classes to be added.